src/core/components/link/Link.stories.tsx

```diff
@@ -0,0 +1,296 @@
+
+export const PrimaryLinkBrandAltTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkBrandAlt}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+PrimaryLinkBrandAltTheme.args = {
+	icon: undefined,
+};
+PrimaryLinkBrandAltTheme.parameters = {
+	backgrounds: {
+		default: 'brandAlt',
+		values: [storybookBackgrounds.brandAlt],
+	},
+};
+asChromaticStory(PrimaryLinkBrandAltTheme);
+
+// *****************************************************************************
+
+export const SecondaryLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+SecondaryLinkLightTheme.args = {
+	priority: 'secondary',
+	icon: undefined,
+};
+asChromaticStory(SecondaryLinkLightTheme);
+
+// *****************************************************************************
+
+export const PrimarySubduedLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+PrimarySubduedLinkLightTheme.args = {
+	subdued: true,
+	icon: undefined,
+};
+asChromaticStory(PrimarySubduedLinkLightTheme);
+
+// *****************************************************************************
+
+export const SecondarySubduedLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+SecondarySubduedLinkLightTheme.args = {
+	priority: 'secondary',
+	subdued: true,
+	icon: undefined,
+};
+asChromaticStory(SecondarySubduedLinkLightTheme);
+
+// *****************************************************************************
+
+export const PrimaryIconLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+asChromaticStory(PrimaryIconLinkLightTheme);
+
+// *****************************************************************************
+
+export const SecondaryIconLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+SecondaryIconLinkLightTheme.args = {
+	priority: 'secondary',
+};
+asChromaticStory(SecondaryIconLinkLightTheme);
+
+// *****************************************************************************
+
+export const SubduedIconLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+SubduedIconLinkLightTheme.args = {
+	subdued: true,
+};
+asChromaticStory(SubduedIconLinkLightTheme);
+
+// *****************************************************************************
+
+export const RightIconLinkLightTheme = (args: LinkProps) => (
+	<ThemeProvider theme={linkDefault}>
+		<Template {...args} />
+	</ThemeProvider>
+);
+RightIconLinkLightTheme.args = {
+	iconSide: 'right',
+};
+asChromaticStory(RightIconLinkLightTheme);
+
+// *****************************************************************************
+
+// export const PriorityLight = (_: LinkProps) => (
```

Comment:
do we need to keep this stuff?

Reply:
Whoops🤦‍♂️  Nope, I'll remove that

Reply:
Done in https://github.com/guardian/source/pull/910/commits/112d4cef13f3484cbac81f7554c120094cc61cfe